modules/rosa-sts-aws-requirements.adoc

```diff
@@ -21,6 +21,99 @@ The following prerequisites must be complete before you deploy a {product-title}
 Customers are encouraged, but not mandated, to deploy resources in a Virtual Private Cloud (VPC) separate from the VPC hosting {product-title} and other Red Hat supported services.
 ====
 
+//
+// Start work for OSDOCS-3198
+//
+[id="rosa-associating-account_{context}"]
+== Associating your AWS account
```

Comment:
As written, this part seems part of the pre-reqs for any ROSA deployment.  We should be clear that "Associating your AWS account" is only if you plan to use the OCM UI.  This is not required if you plan to use the CLI only.